### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by hq-nguyen**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by hq-nguyen — COMMENTED** [review]
good one

**Review by hai-quan3 — COMMENTED** [review]
on the right foot

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (hq-nguyen) [inline index.html:50]: add button to play again instead of refreshing

Comment (hai-quan3) [reply index.html:50]: done

File: README.md [hunk 2]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
...
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 5 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
+## Step 5: Apply suggested changes
```
Comment (hq-nguyen) [inline README.md:22]:
```suggestion
## Step 5: Applied suggested changes
```

File: README.md [hunk 3]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
...
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 5 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
+## Step 5: Applied suggested changes
```
Comment (hq-nguyen) [inline README.md:22]:
```suggestion
## Have a nice day!
```

**Pull request merged**